utils/build-script-impl

```diff
@@ -3138,7 +3143,7 @@ for host in "${ALL_HOSTS[@]}"; do
                 if [[ -z "${INSTALL_LLBUILD}" ]] ; then
                     continue
                 fi
-                INSTALL_TARGETS=install-swift-build-tool
+                INSTALL_TARGETS="install-swift-build-tool install-libllbuildSwift"
```

Comment:
Let's start by not actually installing these bindings, just getting them to build.

I'm not even sure we ultimately want to install them, versus building them into a .a we can use.

Reply:
I'm worried that just using the `.a` will make it difficult to update the llbuild library. And if we don't install them, what's the next for me if I want to advance on the front of using the llbuild library in SwiftPM?

Reply:
You could still use the library in SwiftPM, you just wouldn't be able to only build it from a toolchain.

I'm fine with starting with a dylib, but I still think we should separate the build and installation changes.